### ISSUE TITLE
Folder upload not working

### ISSUE THREAD

**Issue by Pilskalns** [ashkulz/NppFTP#18]
Folder upload is not working:

![error](https://cloud.githubusercontent.com/assets/6931156/6555939/84fc5e06-c672-11e4-8369-86dd83fd65e6.JPG)

On FileZilla it works.

**Comment by ashkulz** [comment]
This is not a general purpose FTP client, so you should use a standalone client for this.

**Comment by mfinky** [comment]
Hello Ashish.

I only wanted to share my personal experience, in case it helps someone else.

1)      I REALLY  enjoy nppftp.  I fint it is an Excellent Tool for me when I need to work maintaining websites.    Once more, congrats and thank you.

2)      Sometimes, I need to upload a JPG file to a website, and … well… (npp + nppftp) wouldn’t be the ‘official’ way to do it … but to avoid using another ftp client tool (for instance that would be fireFtp, used inside FFox) I decided to give this a try:

a.      I opened the jpg within npp. Of course I see garbage on screen

b.      I uploaded that file to the desired folder.

c.       Closed the tab containing the jpg file

The trick worked wonderfully.
